src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsRequest.js

```diff
@@ -0,0 +1,137 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import {hide as hideSidebar} from '../../../../libs/actions/Sidebar';
+import withWindowDimensions, {
+    windowDimensionsPropTypes,
+} from '../../../../components/withWindowDimensions';
+import {getNewChatOptions} from '../../../../libs/OptionsListUtils';
+
+import OptionsSelector from '../../../../components/OptionsSelector';
+import ONYXKEYS from '../../../../ONYXKEYS';
+
+const personalDetailsPropTypes = PropTypes.shape({
+    // The login of the person (either email or phone number)
+    login: PropTypes.string.isRequired,
+
+    // The URL of the person's avatar (there should already be a default avatar if
+    // the person doesn't have their own avatar uploaded yet)
+    avatar: PropTypes.string.isRequired,
+
+    // This is either the user's full name, or their login if full name is an empty string
+    displayName: PropTypes.string.isRequired,
+});
+
+const propTypes = {
+    // Callback to inform parent modal of success
+    onStepComplete: PropTypes.func.isRequired,
+
+
+    // All of the personal details for everyone
+    personalDetails: PropTypes.objectOf(personalDetailsPropTypes).isRequired,
+
+    // All reports shared with the user
+    reports: PropTypes.shape({
+        reportID: PropTypes.number,
+        reportName: PropTypes.string,
+    }).isRequired,
+
+
+    ...windowDimensionsPropTypes,
+};
+
+class IOUParticipantsRequest extends Component {
+    constructor(props) {
+        super(props);
+        this.addSingleParticipant = this.addSingleParticipant.bind(this);
+
+        const {personalDetails, userToInvite} = getNewChatOptions(
+            props.reports,
+            props.personalDetails,
+            '',
+        );
+
+        this.state = {
+            personalDetails,
+            userToInvite,
+            searchValue: '',
+        };
+    }
+
+    /**
```

Comment:
```suggestion
   /**
```

Try to make these line up like this:

```
/**
 *
 *
```